doc/developers/advanced_installation.rst

```diff
@@ -59,7 +59,7 @@ feature, code or documentation improvement).
 #. Install Cython_ and build the project with pip in :ref:`editable_mode`::
 
         pip install cython
```

Comment:
is this not handled by `project.toml` now?

Reply:
yes and no. By default, pip will build sklearn in an isolated env where it automatically installs cython. But sklearn will be re-compiled entirely each time with `pip install -e .`

So the preferable way is to install the build dependencies in your env and run `pip install --no-build-isolation -e .`

Reply:
I'm a bit confused now, does that mean `pip install --no-build-isolation -e .` ignores `pyproject.toml`?

Reply:
yes